### ISSUE TITLE
[Ubuntu 14.04] Brackets 1.1 shows empty folders and has problems opening files

### ISSUE THREAD

**Issue by Oreqizer** [adobe/brackets#10651]
![2015-02-24-084307_1920x1080_scrot](https://cloud.githubusercontent.com/assets/11172408/6345089/d59ebb18-bc01-11e4-88f4-161c8b58797a.png)
![2015-02-24-084530_1920x1080_scrot](https://cloud.githubusercontent.com/assets/11172408/6345092/d97f649e-bc01-11e4-883a-e6352587cf31.png)

OS: Elementary OS 0.3 Freya (64bit) - build on Ubuntu 14.04
Brackets: 1.1.0-15558

My Brackets basically has two issues. First, it won't open the index.html starter file, for some reason. Another one is, that it does not read folders - always shows them as empty, although there are files inside. Files are owned by me, I checked.

Also, it does not remember, where I left off - always opens Getting Started tab.

Do you guys have any suggestions?

Thanks!

**Comment by peterflynn** [comment]
@Oreqizer Some questions:

* Where did you get this build of Brackets from?  Is it an official build from http://brackets.io?
* What filesystem is Brackets installed on?  What about the other folders you're opening?  (You can check via `df -T <path>`).
* Check the file `~/.config/brackets/state.json`. Does if have correct permissions? What is the content of the file currently?

**Comment by Oreqizer** [comment]
FIXED: i got some other issues with the OS (it's still beta). Reinstalled the enitre OS and that fixed the issue - looks like i accidentally reinstalled some essential libraries.

TL;DR: OS issue, not brackets - runs fine now.

Thanks, though!

**Comment by Oreqizer** [comment]
BROKEN AGAIN:
After I restarted the system, the problem appeared again, together with all other problems - this is the OS issue, though. I will head over to their support and report.

**Comment by colin-dumitru** [comment]
Can confirm, using brackets version which was downloaded today.

OS: Ubuntu 12.04
FS: Ext4
Version: Brackets.1.1.Extract.64-bit.deb

**Comment by peterflynn** [comment]
@colin-dumitru Please file a new bug -- the bug above was due to problems with the beta version of "Elementary OS."

**Comment by peterflynn** [comment]
Closing since the problem appears to be with the beta OS in this case.  @Oreqizer let me know if you find information that suggests Brackets is at fault here...
